### DISCUSSION TITLE
Clicking within fill:none object in IconLayer

### DISCUSSION THREAD

**Discussion by simonwarchol** [visgl/deck.gl#8028] [Q&A] [answered]
I use the following iconLayer to superimpose a circle on my deckgl. 
```
const iconSvg = `
  <svg width="1000" height="1000" viewBox="0 0 1000 1000"  xmlns="http://www.w3.org/2000/svg">
      <circle cx="500" cy="500" r="497" fill="rgba(0,0,0,0.5)" stroke="#ffffff"  stroke-width="6"/>
    </svg>
  `

  const circleOverlay = new IconLayer({
    id: 'line-layer-#detail#',
    data: [lensPosition],
    getIcon: () => ({
      url: `data:image/svg+xml;charset=utf-8,${encodeURIComponent(iconSvg)}`,
      width: 1000,
      height: 1000
    }),
    sizeScale: 2,
    getSize: d => 100,
    getPosition: d => d,
    onClick: (info, event) => {
      console.log(info, event)
    },
    pickable: true,
  });
```

As is, i need to have a fill opacity of at least `0.05` to register click events within the circle. Is there any way to register these events within the circle without marginally distorting the interior color?

I've tried to add `pointer-events="fill"` / `pointer-events="none"` / `pointer-events="visiblePainted" ` on the circle to no avail.

This circle will need to become a more complex shape, so using scatterplotLayer isn't an option.

**Comment by Pessimistress** [comment, marked as answer]
[alphaCutoff](https://deck.gl/docs/api-reference/layers/icon-layer#alphacutoff)

**Reply by simonwarchol** [reply]
Amazing -- thank you!
